airflow/templates/etc/_airflow.cfg.tpl

```diff
@@ -0,0 +1,236 @@
+# SequentialExecutor, LocalExecutor, CeleryExecutor
+{{ if not .default.core.executor }}#{{ end }}executor = {{ .default.core.executor | default "CeleryExecutor" }}
+
+# The SqlAlchemy connection string to the metadata database.
+# SqlAlchemy supports many different database engine, more information
+# their website
+{{ if not .default.core.sql_alchemy_conn }}#{{ end }}sql_alchemy_conn = {{ .default.core.sql_alchemy_conn | default "postgresql+psycopg2://airflow:airflow@postgresql/airflow" }}
+
+# The SqlAlchemy pool size is the maximum number of database connections
+# in the pool.
+{{ if not .default.core.sql_alchemy_pool_size }}#{{ end }}sql_alchemy_pool_size = {{ .default.core.sql_alchemy_pool_size | default "5" }}
+
+# The SqlAlchemy pool recycle is the number of seconds a connection
+# can be idle in the pool before it is invalidated. This config does
+# not apply to sqlite.
+{{ if not .default.core.sql_alchemy_pool_recycle }}#{{ end }}sql_alchemy_pool_recycle = {{ .default.core.sql_alchemy_pool_recycle | default "3600" }}
+
+# The amount of parallelism as a setting to the executor. This defines
+# the max number of task instances that should run simultaneously
+# on this airflow installation
+{{ if not .default.core.parallelism }}#{{ end }}parallelism = {{ .default.core.parallelism | default "32" }}
+
+# The number of task instances allowed to run concurrently by the scheduler
+{{ if not .default.core.dag_concurrency }}#{{ end }}dag_concurrency = {{ .default.core.dag_concurrency | default "16" }}
+
+# Are DAGs paused by default at creation
+{{ if not .default.core.dags_are_paused_at_creation }}#{{ end }}dags_are_paused_at_creation = {{ .default.core.dags_are_paused_at_creation | default "False" }}
+
+# The maximum number of active DAG runs per DAG
+{{ if not .default.core.max_active_runs_per_dag }}#{{ end }}max_active_runs_per_dag = {{ .default.core.max_active_runs_per_dag | default "16" }}
+
+# Whether to load the examples that ship with Airflow. It's good to
+# get started, but you probably want to set this to False in a production
+# environment
+{{ if not .default.core.load_examples }}#{{ end }}load_examples = {{ .default.core.load_examples | default "False" }}
+
+# Where your Airflow plugins are stored
+{{ if not .default.core.plugins_folder }}#{{ end }}plugins_folder = {{ .default.core.plugins_folder | default "/usr/local/airflow/plugins" }}
+
+# Secret key to save connection passwords in the db
+{{ if not .default.core.fernet_key }}#{{ end }}fernet_key = {{ .default.core.fernet_key | default "fKp7omMJ4QlTxfZzVBSiyXVgeCK-6epRjGgMpEIsjvs=" }}
+
+# Whether to disable pickling dags
+{{ if not .default.core.donot_pickle }}#{{ end }}donot_pickle = {{ .default.core.donot_pickle | default "False" }}
+
+# How long before timing out a python file import while filling the DagBag
+{{ if not .default.core.dagbag_import_timeout }}#{{ end }}dagbag_import_timeout = {{ .default.core.dagbag_import_timeout | default "30" }}
+
+[webserver]
```

Comment:
everything in this section references .default.web.foo, while in your values file uses 'webserver' as a key. rename everything in this section to use .default.webserver.foo instead of .default.web.foo